CODEOWNERS

```
@@ -1 +1 @@
-* @giovannt0 @hacktobeer @tomchop @sa3eed3ed
+* @libcloudforensics
```

```suggestion
* @google/libcloudforensics
```
I think this should be the proper format: @org/team-name as in https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners#codeowners-syntax